### PR TITLE
will render an html styleguide

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: lib/styleguide.template.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+'use strict';
+const _ = require('lodash');
```
Comment (jgallen23) [inline lib/styleguide.template.js]: FYI. all the lodash functions you used are built into node

File: lib/styleguide.template.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+module.exports = (cssVars) => {
+return `
+<html>
+  <head> </head>
+  <body>
+    <h1> Styleguide </h1>
+    <h2> Colors <h2>
+    <table>
+      ${_.reduce(cssVars, (memo, value, name) => {
```
Comment (jgallen23) [inline lib/styleguide.template.js]: would a map work better?

Comment (jgallen23) [inline lib/styleguide.template.js]: nm, you have the if statement

File: tasks/css.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 const fs = require('fs');
 const path = require('path');
-
+const _ = require('lodash');
```
Comment (jgallen23) [inline tasks/css.js]: you added lodash, but aren't using it.  make sure to lint!

File: tasks/css.js [hunk 4]
```diff
@@ -139,12 +139,27 @@ class CssTask {
     log(`Wrote: ${this.input} → ${output}`);
     log(`Wrote: ${this.input}.map → ${output}.map`);
   }
+
+  exportStyleguide(inputName, outputName) {
+    try {
+      const htmlTemplate = require(path.join(process.cwd(), 'lib', 'styleguide.template.js'));
```
Comment (jgallen23) [inline tasks/css.js]: this should be __dirname, not cwd()

File: tasks/css.js [hunk 5]
```diff
@@ -139,12 +139,27 @@ class CssTask {
...
+      const output = htmlTemplate(this.cssVars);
+      fs.writeFileSync(outputName, output);
+    } catch (e) {
+        console.log(e)
```
Comment (jgallen23) [inline tasks/css.js]: `log(['error']...`

Comment (orthagonal) [inline tasks/css.js]: logr 1.10.0 will add an 'error' tag when you pass it an Error object  :-)

**Pull request closed without merging**